site/en/r2/guide/migration_guide.ipynb

```diff
@@ -558,8 +558,7 @@
         "* Collect layer parameters in `__init__`.\n",
         "* Build the variables in `build`.\n",
         "* Execute the calculations in `call`, and return the result.\n",
-        "\n",
-        "The `tf.variable_scope` is essentially a layer of its own. So rewrite it as a `tf.keras.layers.Layer`. See [the guide](keras/custom_layers_and_models.ipynb) for details."
```

Comment:
Keep this, please

Reply:
I've reverted this commit but isn't it duplicate?